swo_aws_extension/flows/steps/setup_context.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from mpt_extension_sdk.mpt_http.mpt import update_order
+
+from swo_aws_extension.aws.client import AWSClient
+from swo_aws_extension.aws.config import Config
+from swo_aws_extension.constants import PhasesEnum
+from swo_aws_extension.flows.order import InitialAWSContext
+from swo_aws_extension.flows.steps.base import BasePhaseStep
+from swo_aws_extension.flows.steps.errors import ConfigurationStepError, SkipStepError
+from swo_aws_extension.parameters import get_phase, set_phase
+
+logger = logging.getLogger(__name__)
+
+
+class SetupContext(BasePhaseStep):
+    """Initial setup context step."""
+
+    def __init__(self, config: Config, role_name: str) -> None:
+        self._config = config
+        self._role_name = role_name
+
+    def pre_step(self, context: InitialAWSContext) -> None:
+        """Validate if step should run."""
+        if not context.pm_account_id:
+            raise ConfigurationStepError(
+                "SetupContextError - PMA account is required to setup AWS Client in context"
+            )
+
+    def process(self, context: InitialAWSContext) -> None:
+        """Setup AWS client in context."""
```

Comment:
Umm...I believe we should change the class docstrings to -> """Setup AWS client in context"""
```suggestion
    @override
    def process(self, context: InitialAWSContext) -> None:
```